api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -1095,6 +1101,42 @@ public void triggerStateConversion_shouldTestTransitionToState(){
 		pwsi.triggerStateConversion(patient, trigger, dateConverted);
 		assertEquals(patientProgram.getStates().size(), (patientStatesSize + 1));
 	}
+
+	/**
+	 * Test to check the implementation of Patient Program Attribute Type by Uuid
```

Comment:
Given the method name, do would these comments add any value? 😊 
https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/

Reply:
I really think so, since methods generally have a maximum number of characters in their names, it is possible to have a name that does not fully explain the purpose of the method, so it is good to have a **javadoc** to give more details if they exist, especially when the method is complex.
However, if the practice is not valid, they may be removed.

Reply:
Ofcourse there will always be cases where we need comments. We just cannot completely get rid of them. 😊 
But for these particular methods of yours, do they add any value?

Reply:
I just answered that, but after we agreed on the previous comment, makes me feel like they don't add any value. Correct?

Reply:
In that case, would we keep them?

Reply:
No! I will remove them 😏